gatk-launch

```diff
@@ -94,17 +96,20 @@ def main(args):
             print("")
             print("   --sparkRunner <target>    controls how spark tools are run")
             print("     valid targets are:")
-            print("     LOCAL:   run using the in-memory spark runner")
-            print("     SPARK:   run using spark-submit on an existing cluster ")
-            print("              --sparkMaster must be specified")
-            print("              --sparkSubmitCommand may be specified to control the Spark submit command")
-            print("              arguments to spark-submit may optionally be specified after -- ")
-            print("     GCS:     run using Google cloud dataproc")
-            print("              commands after the -- will be passed to dataproc")
-            print("              --cluster <your-cluster> must be specified after the --")
-            print("              spark properties and some common spark-submit parameters will be translated to dataproc equivalents")
+            print("     LOCAL:      run using the in-memory spark runner")
+            print("     SPARK:      run using spark-submit on an existing cluster ")
+            print("                 --sparkMaster must be specified")
+            print("                 --sparkSubmitCommand may be specified to control the Spark submit command")
+            print("                 arguments to spark-submit may optionally be specified after -- ")
+            print("     GCS:        run using Google cloud dataproc")
+            print("                 commands after the -- will be passed to dataproc")
+            print("                 --cluster <your-cluster> must be specified after the --")
+            print("                 spark properties and some common spark-submit parameters will be translated ")
+            print("                 to dataproc equivalents")
             print("")
-            print("   --dryRun    may be specified to output the generated command line without running it")
+            print("   --dryRun      may be specified to output the generated command line without running it")
+            print("   --javaOptions 'OPTION1=X[, OPTION2=Y ... ]'   optional - pass the given string of options to the ")
+            print("                 java JVM at runtime (for local runs only)")
```

Comment:
Remove `(for local runs only)` once you extend this feature to the Spark modes (see comment below).